docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -12,10 +12,11 @@ This hardening guide is intended to be used for RKE clusters and is associated w
 
 | Rancher Version | CIS Benchmark Version | Kubernetes Version           |
 |-----------------|-----------------------|------------------------------|
-| Rancher v2.7    | Benchmark v1.23       | Kubernetes v1.23 up to v1.25 |
+| Rancher v2.7    | Benchmark v1.7       | Kubernetes v1.24 up to v1.25 |
```

Comment:
Any reason why we are removing the versions < 1.25 from the hardening guide? I didn't get this change.
Rancher v2.7 still supports K8s v1.23 to v1.24, so we need to maintain them there.
Ideally the change should be something like this, unless I'm missing some context:

```suggestion
| Rancher v2.7    | Benchmark v1.23       | Kubernetes v1.23             |
| Rancher v2.7    | Benchmark v1.24       | Kubernetes v1.24             |
| Rancher v2.7    | Benchmark v1.7        | Kubernetes v1.25 up to v1.26 |
```